scalding-parquet-scrooge/src/main/java/com/twitter/scalding/parquet/scrooge/ParquetScroogeOutputFormat.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2012 Twitter, Inc.
```

Comment:
actually, I don't think we need to keep adding copyright notices. Seems like make work, and the repo LICENSE should be enough (I think @travisbrown signed off on this statement).